entgql/pagination.go

```diff
@@ -84,6 +84,60 @@ func (o OrderDirection) Reverse() OrderDirection {
 	return OrderDirectionDesc
 }
 
+// NullsOrderDirection defines the directions in which to order the Nulls of a list of items.
+type NullsDirection string
+
+const (
+	// NullsOrderDirectionFirst specifies nulls to be ordered first
+	NullsFirst NullsDirection = "First"
```

Comment:
Since this is an enum it should be uppercase, that would also match the order direction formatting.